pkg/config/system_probe.go

```diff
@@ -169,7 +169,11 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "dest_excludes"), map[string][]string{})
 
 	// network_config namespace only
+
+	// For backward compatibility
 	cfg.BindEnv(join(netNS, "enable_http_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTP_MONITORING")
+	cfg.BindEnv(join(smNS, "enable_http_monitoring"))
+
 	cfg.BindEnv(join(netNS, "enable_https_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTPS_MONITORING")
```

Comment:
Do we want to move `enable_https_monitoring` as well?

Reply:
We decided to do a first PR with a single value, create all helper functions and understand the whole ecosystem, and after that move the rest of the features